### ISSUE TITLE
gnome canvas is deprecated

### ISSUE THREAD

**Issue by benibela** [jneem/homespring#2]
Debian Buster [has deprecated libgnome](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=885677), so lablgnomecanvas  is no longer included in liblablgtk2-gnome-ocaml

As consequence hsgui does not work anymore, without copying the ocaml gnomecanvas files from an older Debian version

Hsgui needs to be updated to GTK3 or something

**Comment by jneem** [comment]
Indeed. Unfortunately, there don't seem to be any maintained ocaml bindings to GTK3. Perhaps a reasonable short-term fix would be to make compiling hsgui optional.
